docs/integration/RequestResponse.mdx

```diff
@@ -0,0 +1,120 @@
+---
+sidebar_position: 2
+---
+
+# Request-Response processing mode only
+
+## Overview
+
+Depending on your network topology after you deploy a `Request-response` scenario, there should be either a k8s service or ingress allowing you to connect to scenario.
+
+## Connection
+
+In case of `service` you can trigger your scenario with example curl command:
+
+```bash
+curl -X POST -d "payload_based_on_input_schema" 'http://<scenario_name>'
+```
+
+For `ingress` configuration it looks quite similar:
+
+```bash
+curl -X POST -d "payload_based_on_input_schema" 'http://<ingress_domain><scenario_name>'
+```
+
+## OpenAPI
+
+After you deploy your `Request-Response` scenario an OpenAPI definition based on input and output schema you have provided in scenario properties will be available.
```

Comment:
Request-response unfortunately (this is the agreement, at least for now)

What about taking this to your stand up? I also prefer "Request-Response".